tests/controllers/coursesController.test.js

```diff
@@ -5,63 +5,88 @@ require('dotenv').config();
 const coursesController = require('../../src/controllers/coursesController');
 const ConflictError = require('../../src/errors/ConflictError');
 const NotFoundError = require('../../src/errors/NotFoundError');
+const Course = require('../../src/models/Course');
 
 jest.mock('../../src/models/Course.js');
 
 describe('POST /course', () => {
```

Comment:
Esse teste seria unitário, porem seu describe esta se referindo a rota. Em testes unitários nossos describes podem se referir a função e cada it seria um teste nessa função.